Document Tempo monitor tab and span RED metrics (#11)
max-cx/openshift-docs: modules/distr-tracing-otel-config-collector.adoc

Exporters:: An exporter, which can be push or pull based, is how you send data to one or more back ends or destinations. By default, no exporters are configured. One or more exporters must be configured. Exporters can support one or more data sources. Exporters might be used with their default settings, but many exporters require configuration to specify at least the destination and security settings.

> Connectors:: A connector is both an exporter and receiver. As the name suggests a Connector connects two pipelines: It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It may consume and emit data of the same data type, or of different data types. A connector may generate and emit data to summarize the consumed data, or it may simply replicate or route data.

Reviewer: Some of the info is repeated in another sentence. I
```suggestion
Connectors:: A connector connects two pipelines: It consumes data as an exporter at the end of one pipeline and emits data as a receiver at the start of another pipeline. It can consume and emit data of the same or different data type. It can generate and emit data to summarize the consumed data, or it can merely replicate or route data.
```

<4> The client password is configured as a client authenticator for the BasicAuth extension.
<5> The authenticator configuration can be assigned to an OTLP receiver.
<6> The authenticator configuration can be assigned to an OTLP exporter.

[id="connectors_{context}"]
=== Connectors

[id="spanmetrics-connector_{context}"]
==== Spanmetrics connector

> Aggregates Request, Error and Duration (R.E.D) OpenTelemetry metrics from span data.

Reviewer: Serial comma:
```suggestion
Aggregates Request, Error, and Duration (R.E.D) OpenTelemetry metrics from span data.
```
